core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySupport.scala

```diff
@@ -271,13 +271,13 @@ trait DataRepoBigQuerySupport extends LazyLogging {
 
   /**
    * Translates a BigQuery result set into the pagination metadata that Rawls expects.
-   * @param queryResults the BigQuery result set
    * @param entityQuery the query criteria supplied by the user, which includes page size
    * @return the Rawls-flavor pagination metadata
    */
-  def queryResultsMetadata(queryResults:TableResult, totalRowCount: Int, entityQuery: EntityQuery): EntityQueryResultMetadata = {
+  def queryResultsMetadata(totalRowCount: Int, entityQuery: EntityQuery): EntityQueryResultMetadata = {
     val pageCount = Math.ceil(totalRowCount.toFloat / entityQuery.pageSize).toInt
-    EntityQueryResultMetadata(totalRowCount, queryResults.getTotalRows.toInt, pageCount)
+    // we don't support filtering in BQ, so unfilteredCount and filteredCount are the same
+    EntityQueryResultMetadata(totalRowCount, totalRowCount, pageCount)
```

Comment:
sigh I got it wrong in the last PR; both `filteredCount` and  `unfilteredCount` should reflect the total number of rows, as supplied by TDR, for the given entity type.